docs/docs/policies.mdx

```diff
@@ -511,32 +537,32 @@ conversation just before each action was taken.
 
 Every event in a trackers history creates a new state (e.g. running a bot
 action, receiving a user message, setting slots). Featurizing a single state
-of the tracker has a couple steps:
+of the tracker has two steps:
 
 1. **Tracker provides a bag of active features**:
 
-       * features indicating intents and entities, if this is the first
-         state in a turn, e.g. it's the first action we will take after
-         parsing the user's message. (e.g.
-         `[intent_restaurant_search, entity_cuisine]` )
+    * features indicating intents and entities, if this is the first
+     state in a turn, e.g. it's the first action we will take after
+     parsing the user's message. (e.g.
+     `[intent_restaurant_search, entity_cuisine]` )
 
-       * features indicating which slots are currently defined, e.g.
-         `slot_location` if the user previously mentioned the area
-         they're searching for restaurants.
+    * features indicating which slots are currently defined, e.g.
+     `slot_location` if the user previously mentioned the area
+     they're searching for restaurants.
 
-       * features indicating the results of any API calls stored in
-         slots, e.g. `slot_matches`
+    * features indicating the results of any API calls stored in
+     slots, e.g. `slot_matches`
 
-       * features indicating what the last bot action or bot utterance was (e.g.
-         `prev_action_listen`)
+    * features indicating what the last bot action or bot utterance was (e.g.
+     `prev_action_listen`)
 
-       * features indicating if any loop is active and which one
+    * features indicating if any loop is active and which one
```

Comment:
Sadly this is not rendering well. I think it is supposed to work with the latest update to the theme -- but i think you want the asterisks to be on the same indentation as the first asterisk above (currently one space in)

Reply:
ok let me see

Reply:
just kidding, i think i hadn't installed the latest update 🙂

Reply:
ahah! then greaaaaat 💯